internal/static/setup.go

```diff
@@ -0,0 +1,37 @@
+package static
+
+import (
+	"time"
+
+	"tcp-load-balancer/internal/config"
+	"tcp-load-balancer/internal/server"
+	"tcp-load-balancer/internal/upstream"
+)
+
+// Setup configures upstream hosts and downstream clients to demonstrate functionality.
+func Setup(l *server.LoadBalancer, numberOfHosts int, numberOfClients int, clientMessageInterval time.Duration) error {
+
+	// TODO: Implement a non-static method for registering upstream hosts (outside the scope of this challenge).
+	if err := RegisterUpstreamHosts(l, numberOfHosts); err != nil {
+		return err
+	}
+
+	// TODO: Implement a non-static method for connecting clients (outside the scope of this challenge).
+	InitializeHelloClients(l.Address().String(), clientMessageInterval, numberOfClients)
+
+	return nil
+}
+
+// RegisterUpstreamHosts adds n static hosts to the load balancer for testing and demonstration purposes.
+func RegisterUpstreamHosts(l *server.LoadBalancer, n int) error {
```

Comment:
That's correct. Everything in the `static` package is only for illustrating LB usage -- the entire package could be removed at some point. I'll add a note to make that more clear.

I'm not sure what you mean by making the functions "unexpected" is that like putting them in a `_test` package? Or just separating them into a different process so that our main.go doesn't have to import them?

Reply:
Ah, got it. Thanks for clarifying. I agree, that would be clearer to read. 

I've moved it under the `/test` folder instead of `/internal/package`. 

I could also see reasons for moving it to `/examples` instead of test in the long-term, but it would take some updates to make it more of an external example of how other clients can connect, rather than an internal test tool.

Reply:
To your point about it not leveraging `/internal`, I've also added this TODO at the top of the `test` package:
```
////////////////////////////////////////////////////////////////////////////////////////////////////
// This package is used entirely for demonstration purposes (to statically configure hosts/clients).
// In the future, it should be modified so that it can be used as an example (and moved to /examples)
// for other clients to use this library. Leaving that out of scope for this challenge.
////////////////////////////////////////////////////////////////////////////////////////////////////
```

Let me know if you think we should include the example setup (leveraging purely exported functions rather than the internal stuff this package uses now) within scope for this project.